components/layout/constants.js

```diff
@@ -0,0 +1,37 @@
+
+export const SIDER_LINKS = [
+	{
+		icon: 'dashboard',
+		key: 'home',
+		name: 'Home',
+		path: '/admin',
+	},
+	{
+		icon: 'appstore',
+		key: 'resources',
+		name: 'Learning Resources',
+		path: '/admin/resources',
+	},
+	{
+		icon: 'edit',
+		key: 'practice',
+		name: 'Practice',
+		path: '/admin/practice',
+	},
+	{
+		icon: 'thunderbolt',
+		key: 'challenge',
+		name: 'Challenge Friends',
+		path: '/admin/challenge',
+	},
+	{
+		icon: 'user',
+		key: 'profile',
+		name: 'Profile',
+		path: '/admin/profile',
+	}
+];
+
+export const LOGO_TEXT = 'Leer';
+
+export const greetUser = user => `Hello ${user || ''}`;
```

Comment:
1. do not put functions in `constants.js` files., only primitives or collections of primitives. If you have reusable functions, you can create a `utils` folder on the root and save your functions there.
2. delete this function, because it is not doing really. Just use `Hello ${user || ''}` directly in the code